### PR TITLE
Added Docs tasks.

### PR DESCRIPTION
**Pull request by DamianMullins**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -251,7 +251,15 @@ Runs the [`assemble`](#assemble) task when documentation files are changed.
 
 - #### `docs`
 
-This will build a fresh copy of any documentation found in the `docs` directory using Assemble, then call the [`watch`](#watch) task which will watch for any file changes, and finally call the [`browser-sync:docs`](#browser-syncdocs) task which reloads the web page when changes are detected in the `docs/dist` directory.
+Builds a fresh copy of any documentation found in the [`config.docs.rootDir`](#rootdir) directory using Assemble, then watch for any file changes and reload the web page when changes are detected in the [`config.docs.distDir`](#distdir) directory.
```
Comment (ashleynolan) [inline README.md]: Grammar alert 🚨  Should be '…then **watches** for any file changes and **reloads**…', to match the tense of the previous part of the sentence.

File: README.md [hunk 2]
```diff
@@ -251,7 +251,15 @@ Runs the [`assemble`](#assemble) task when documentation files are changed.
...
+
+- #### `docs:deploy`
+
+Build the documentation and then pushes the dist directory to the gh-pages branch.
```
Comment (ashleynolan) [inline README.md]: Should be `Builds` or just `push` so that the tenses match.

File: tasks-dev/assemble.js [hunk 3]
```diff
@@ -53,7 +52,6 @@ gulp.task('assemble', () => {
         //     ext: '.html'
         // }))
 
-        .pipe(debug())
```
Comment (ashleynolan) [inline tasks-dev/assemble.js]: Does removing this mean that the task won’t now output what files are being changed?

Comment (DamianMullins) [inline tasks-dev/assemble.js]: It still displays `❯❯ File: index.hbs was changed`.

**Pull request merged**